### PR TITLE
i18n(zh-cn): update `getting-started` (v2 release) 

### PR DESCRIPTION
**Pull request by manchan4869**
#### What kind of changes does this PR include?

- Translated content

#### Description
- update `getting-started` (v2 release)

### CODE REVIEW

File: src/pages/zh-cn/getting-started.mdx [hunk 1]
```diff
@@ -66,7 +77,7 @@ pnpm create astro@latest
 
 📚 探索 Astro 的[运行时 API](/zh-cn/reference/api-reference/)。
```
Comment (delucis) [inline src/pages/zh-cn/getting-started.mdx:78]: Looks like this bullet point and the two above also need updating. Could you update these as part of this PR?

Here’s the current English version:

```md
See examples of some of the key concepts and patterns of an Astro site!

📚 [Add your first page](/en/core-concepts/astro-pages/) to your site.

📚 Read more about Astro’s [project structure](/en/core-concepts/project-structure/).

📚 Learn about Astro's [file-based routing](/en/core-concepts/routing/).

*... find our full API documentation under the **Reference** tab.*
```

**Review by delucis — COMMENTED** [review]
Thanks @manchan4869! Spotted one additional thing we could update here but this looks great!

File: src/pages/zh-cn/getting-started.mdx (hunk 1 above)
Comment (manchan4869) [reply src/pages/zh-cn/getting-started.mdx]: Oh, I missed it. It has been updated now

**Review by delucis — APPROVED** [review]
Thanks for this update @manchan4869! 💜

**Pull request merged**